lib/services/summary.js

```diff
@@ -37,26 +39,30 @@ class SummaryService {
     };
 
     if (this._fullSummary(req) && !decoder.canSummarizeQuickly()) {
+      req.log.info("Making full summary");
+
       req
-      .pipe(decoder)
-      .on('error', onErr)
-      .pipe(new Merger(disk, [], true))
-      .on('error', onErr)
-      .on('end', (layers) => {
-        ok(layers.map((layer) => layer.toJSON()));
-      });
-    } else {
-      decoder.on('data', (_datum) => {
-        decoder.pause();
-        decoder.summarize((err, summary) => {
-          if (err) return res.status(400).send(JSON.stringify(err));
-          return ok(summary);
+        .pipe(decoder)
+        .on('error', onErr)
+        .pipe(new Merger(disk, [], true))
+        .on('error', onErr)
+        .on('end', (layers) => {
+          ok(layers.map((layer) => layer.toJSON()));
         });
-      });
+    } else {
+      req.log.info("Making abbreviated summary");
 
       req
-      .pipe(decoder)
-      .pipe(new DevNull());
+        .pipe(decoder)
+        .on('data', (_datum) => {
+          decoder.pause();
+          decoder.summarize((err, summary) => {
+            if (err) return res.status(400).send(JSON.stringify(err));
+            return ok(summary);
+          });
+        })
+        .on('error', onErr)
+        .pipe(new DevNull());
```

Comment:
Not necessarily related to this PR, but why are we piping things to "DevNull" in the real service.